packages/next-app/src/app/metadata/(components)/OwnedPoolsSidebarItems.tsx

```diff
@@ -72,17 +93,35 @@ function OwnedPoolsSidebarItems({
     );
 
   return (
-    <>
-      {data?.pools.map(
-        (item: ArrElement<GetDeepProp<PoolsWhereOwnerQuery, "pools">>) => (
-          <Link key={item.id} href={`/metadata/${network}/pool/${item.id}`}>
-            <Sidebar.Item isSelected={item.id === poolId}>
-              <PoolCard isSelected={item.id === poolId} pool={item as Pool} />
-            </Sidebar.Item>
-          </Link>
-        )
-      )}
-    </>
+    <div className="flex flex-col gap-y-2 pr-2">
+      <Input
+        label="Search pool"
+        placeholder="Search by name, symbol, id, tokens..."
+        onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
question: doesn't this get typed automatically?

Reply:
This is automatically typed to `FormEvent<HTMLInputElement>` which provides no value on target 

```
Property 'value' does not exist on type 'EventTarget'.
```